server/api/customers.ts

```diff
@@ -16,4 +16,16 @@ customerRouter.get("/",async (req, res, next): Promise<void> => {
     }
 })
 
+// GET /api/customers/:id
+customerRouter.get("/:id", async (req, res, next): Promise<void> => {
+    try {
+        const { id } = req.params;
+        const customer = await getCustomerById(pool, id);
+        const { full_name, email } = customer;
+        res.send( {customer: { id, full_name, email }} );
```

Comment:
nit: will be more extensible if you just strip the password (that way any future fields added to customer will also get added to the response

```suggestion
        delete customer.password;
        res.send({customer});
```